<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlaylistEdit</class>
 <widget class="QDialog" name="PlaylistEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>714</width>
    <height>693</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Playlist Editor</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <item>
    <widget class="QLineEdit" name="nameEdit">
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>The playlist name will be edited here.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListView" name="songView">
     <property name="frameShape">
      <enum>QFrame::Shape::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SelectionMode::ExtendedSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="controlLayout">
     <item>
      <widget class="QPushButton" name="addButton">
       <property name="toolTip">
        <string>Add</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../resources.qrc">
         <normaloff>:/images/dialogs/add.png</normaloff>:/images/dialogs/add.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Remove</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../resources.qrc">
         <normaloff>:/images/dialogs/delete.png</normaloff>:/images/dialogs/delete.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="leftSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="playButton">
       <property name="toolTip">
        <string>Play</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../resources.qrc">
         <normaloff>:/images/player/play.png</normaloff>:/images/player/play.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>addButton</sender>
   <signal>clicked()</signal>
   <receiver>PlaylistEdit</receiver>
   <slot>addSong()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>56</x>
     <y>680</y>
    </hint>
    <hint type="destinationlabel">
     <x>97</x>
     <y>712</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>removeButton</sender>
   <signal>clicked()</signal>
   <receiver>PlaylistEdit</receiver>
   <slot>removeSong()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>108</x>
     <y>680</y>
    </hint>
    <hint type="destinationlabel">
     <x>274</x>
     <y>711</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>songView</sender>
   <signal>activated(QModelIndex)</signal>
   <receiver>PlaylistEdit</receiver>
   <slot>editSong(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>598</x>
     <y>250</y>
    </hint>
    <hint type="destinationlabel">
     <x>748</x>
     <y>143</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playButton</sender>
   <signal>clicked()</signal>
   <receiver>PlaylistEdit</receiver>
   <slot>playSong()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>701</x>
     <y>680</y>
    </hint>
    <hint type="destinationlabel">
     <x>208</x>
     <y>600</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addSong()</slot>
  <slot>removeSong()</slot>
  <slot>editSong(QModelIndex)</slot>
  <slot>playSong()</slot>
 </slots>
</ui>
